### PR TITLE
Added a file about contributing a lesson or for the site

### PR DESCRIPTION
**Pull request by lwjohnst86**
As per [the issue in the council repo](https://github.com/UofTCoders/council/issues/34), I've written up a bit about how to contribute a lesson.

Thoughts and suggestions are greatly welcomed!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+yourself, just recommending something to be done.
+- Submitting a Pull Request from a clone of this repo. This way takes a bit more
+work and requires knowledge of Git and likely HTML. But we would appreciate any
+help! No harm in giving it a try! That's a beauty of using Git, it's hard to
+mess up and break something.
+
+If you want to view the website before submitting a Pull Request to make sure
+your changes are as you expect, you'll need to:
+
+- Install Jekyll by following these [instructions](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/).
```
Comment (QuLogic) [inline CONTRIBUTING.md:167]: You can also view your own fork if using the `gh-pages` branch.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ Welcome to our Mozilla Study Group! A few things to do & know now that you're he
  - **Read the code of conduct:** this Study Group is for everyone - we abide by a [set of rules](https://www.mozillascience.org/code-of-conduct/) that require everyone be treated with respect. Help us make a space where everyone feels welcome, and we'll all have a better time!
  - **Add yourself to the website:** If you'd like to appear on the website under the 'Who we Are' section, have a look at the `_data/members.yml`; send us a pull request with an entry for yourself, or open an issue and we'll do it for you.
 
-## For Organizers
+## For Organizers/Instructors
+
+Check out the CONTRIBUTING [file](https://github.com/UofTCoders/studyGroup/blob/gh-pages/CONTRIBUTING.md) or [page on the site](https://uoftcoders.github.io/studyGroup/CONTRIBUTING) for more details.
```
Comment (QuLogic) [inline README.md]: Link `CONTRIBUTING` too.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+title: Contributing to UofTCoders
+---
+
+## Running a lesson
+
+### Making the lesson
+
+The Mozilla Science Study Group
+handbook [**here**](https://mozillascience.github.io/studyGroupHandbook/lessons.html#reuse)
+and [**here**](http://mozillascience.github.io/studyGroupHandbook/event-types.html#workalong)
```
Comment (QuLogic) [inline CONTRIBUTING.md]: https

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- **Don't use slides and focus on writing the code**
+
+Expanding on this are other things to keep in mind.
+
+**Data**: External data *should not be used* unless it is absolutely necessary. 
+Instead, rely on built-in or sample datasets provided by the respective 
+programming language or package.
+
+**Code**: This is the main thing that should be emphasized. How the code is
+used, it's specific applications, and it's meaning should be the entire focus of
```
Comment (QuLogic) [inline CONTRIBUTING.md]: `its`, in both cases.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Lessons should be submitted as a Pull Request at least *one full day* before the
+session. Submitting the lessons as a Pull Request is also a great way to find
+out whether your lesson matches what is advertised in the Events repo.
+
+### Giving the lesson
+
+A few tips:
+
+- Start from the *very beginning*. Open the program or IDE (RStudio/Jupyter/etc)
```
Comment (QuLogic) [inline CONTRIBUTING.md]: These first two apply to beginner lessons, but I think we do have some intermediate lessons where it's not quite so necessary.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+the lesson. Here are some tips:
+
+- Keep the code as *simple* as possible while still covering the concept. The
+session is, after all, only one hour.
+- When making the code, make *no or few assumptions* about the knowledge of the
+audience.
+- Keep the code *generalizable*. Our members come from diverse fields of
+research. What we share is the need to code.
+
+Lessons should be submitted as a Pull Request at least *one full day* before the
```
Comment (lwjohnst86) [inline CONTRIBUTING.md]: Just so I don't forget, I need to include a mention about using the lesson template files and how to set up a lesson using those files. Probably would be a good idea to update these templates too.. (again, note to self)

**Comment by linamnt** [comment]
I think it looks good, should we just merge it in asap so we can have it available for the new instructors?

**Comment by lwjohnst86** [comment]
@QuLogic @linamnt Mind looking the updated PR over? Then we can merge in :smile:

**Review by linamnt — APPROVED** [review]

**Pull request merged**